src/components/domain/HoldingCard/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { Link } from 'react-router-dom';
+import { css } from '@emotion/css';
+
+import { Card } from 'components/domain';
+import { Button } from 'components/common';
+
+import { useCardDispatch } from 'contexts/CardContextProvider/hooks';
+
+import { PATHS } from 'constants/router';
+import { ICard } from 'types/card';
+
+const CONFIRM_MESSAGE = '정말로 등록된 카드를 삭제하시겠습니까?';
+
+interface HodingCardProps {
+  card: ICard;
+}
+
+function HoldingCard({ card }: HodingCardProps) {
+  const dispatch = useCardDispatch();
+
+  const handleClickDeleteButton = () => {
+    const isCanDeleted = window.confirm(CONFIRM_MESSAGE);
+
+    if (isCanDeleted) {
+      dispatch({ type: 'DELETE_CARD', payload: card });
+    }
```

Comment:
confirm이 주는 `yes or no`는 상당히 명확하므로 인라인으로 작성해주는 것도 좋습니다🤗

```suggestion
    if (window.confirm(CONFIRM_MESSAGE)) {
      dispatch({ type: 'DELETE_CARD', payload: card });
    }
```

그리고 여러 컴포넌트, 페이지 단위에서 dispatch를 호출하는 것보다 커스텀 훅 또는 핸들러를 만들어두고 인자만 넘겨주는 형태로 작성해주는 것이 현재 태영님이 짠 구조에서 좀 더 직관적일 것 같아요.

```ts
// 어딘가에 정의한... 저는 context를 선언한 폴더 내에서 co-location을 염두에 두면 좋겠네요!
export const deleteCard = (card: ICard) => {
  dispatch({ type: 'DELETE_CARD', payload: card });
};

// src/components/domain/HoldingCard/index.tsx
import { deleteCard } from '..//..';

// 생략
    if (window.confirm(CONFIRM_MESSAGE)) deleteCard(card);
```

Reply:
Redux 사용할 때 action을 만드는 형태와 비슷한 것 같네요! 제안해주신 형태가 해당 함수가 무엇을 하려고 하는지 대략적으로 알기에 더 적합한 것 같습니다. 

인성님 말씀처럼 co-location을 염두해두고 action을 모아보도록 하겠습니다. :)